library/swift/test/BUILD

```diff
@@ -52,9 +52,19 @@ envoy_mobile_swift_test(
 )
 
 envoy_mobile_swift_test(
-    name = "stream_client_builder_tests",
+    name = "engine_builder_tests",
     srcs = [
-        "StreamClientBuilderTests.swift",
+        "EngineBuilderTests.swift",
+    ],
+    deps = [
+        "//library/objective-c:envoy_engine_objc_lib",
+    ],
+)
+
+envoy_mobile_swift_test(
```

Comment:
nit: can we alphabetize these test cases?

Reply:
thinking out loud: prolly worth adding a lint/pre-commit check for this in the long run